src/server/lib/middleware/cache.ts

```diff
@@ -8,14 +8,3 @@ export const noCache = (_: Request, res: Response, next: NextFunction) => {
 
   next();
 };
-
-export const removeNoCache = (
-  _: Request,
-  res: Response,
-  next: NextFunction,
-) => {
-  res.removeHeader('Cache-Control');
-  res.removeHeader('Pragma');
-
-  next();
-};
```

Comment:
Now that the `removeNoCache` function is removed we can inline the remaining `noCache` function to the place where it's used (index.ts)?

Reply:
I've made a slightly different improvement, adding `server.use` to src/server/lib/middleware/index.ts and removing it from the individual routes src/server/routes/index.ts, following the middleware pattern in the project https://github.com/guardian/gateway/pull/784/commits/637936a44bddab138d7f1c0d8c88151db5c62d03